test/test_mps.py

```diff
@@ -336,6 +336,7 @@ def mps_ops_modifier(ops):
         'sinc',
         'slice',
         'special.spherical_bessel_j0',
+        'special.xlog1py',
```

Comment:
Special log1p should be tested too?

Reply:
log1p is already implemented using MPS, this is a metal shader implementation. So, it's just an utility (I can make it static if you prefer).